packages/backend/src/apps/m365-excel/actions/index.ts

```diff
@@ -6,12 +6,66 @@ const testAction: IRawAction = {
   description: 'To make excel show up in app list; real actions in later PR.',
   arguments: [
     {
-      label: 'Placeholder',
-      key: 'data',
-      type: 'string' as const,
+      key: 'fileId',
+      label: 'Excel File',
       required: true,
-      description: 'Beep boop',
-      variables: true,
+      description: 'File to edit',
+      type: 'dropdown' as const,
+      variables: false,
+      source: {
+        type: 'query' as const,
+        name: 'getDynamicData' as const,
```

Comment:
nit: actually i see in other files with dynamic data, there is no `as const` used, just wanted to check for consistency sake, is it needed here?